.github/workflows/template.flink-ci.yml

```diff
@@ -286,6 +286,20 @@ jobs:
         if: ${{ !cancelled() && (failure() || !steps.docker-cache.cache.hit) }}
         run: ./tools/azure-pipelines/cache_docker_images.sh save
 
+  flink-kubernetes:
+    name: "Build & test flink-kubernetes with multiple HTTP clients"
+    needs: compile
+    strategy:
+      fail-fast: false
+      matrix:
+        http-client: [ "okhttp", "jdk", "jetty", "vertx" ]
+    uses: ./.github/workflows/template.flink-kubernetes-ci.yml
+    with:
+      http-client: ${{ matrix.http-client }}
```

Comment:
I am curious how long do all these new testing permutations add to the ci times?